RELEASING.md

```diff
@@ -1,12 +1,13 @@
 Releasing
 ========
 
+This repository is set up for autorelease. The process involves pushing a new git tag.
+
  1. Change the version in `gradle.properties` to a non-SNAPSHOT version.
```

Comment:
should always be a `SNAPSHOT` version - we automatically removed the "-SNAPSHOT" string when it's a production release: https://github.com/segment-integrations/analytics-android-integration-comscore/blob/master/gradle/upload.gradle#L30